eva-accession-release/src/test/java/uk/ac/ebi/eva/accession/release/steps/processors/NamedVariantProcessorTest.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright 2019 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.ac.ebi.eva.accession.release.steps.processors;
+
+import org.junit.Test;
+
+import uk.ac.ebi.eva.commons.core.models.IVariant;
+import uk.ac.ebi.eva.commons.core.models.VariantType;
+import uk.ac.ebi.eva.commons.core.models.pipeline.Variant;
+import uk.ac.ebi.eva.commons.core.models.pipeline.VariantSourceEntry;
+
+import static org.junit.Assert.*;
+import static uk.ac.ebi.eva.accession.release.io.VariantMongoAggregationReader.ALLELES_MATCH_KEY;
+import static uk.ac.ebi.eva.accession.release.io.VariantMongoAggregationReader.ASSEMBLY_MATCH_KEY;
+import static uk.ac.ebi.eva.accession.release.io.VariantMongoAggregationReader.CLUSTERED_VARIANT_VALIDATED_KEY;
+import static uk.ac.ebi.eva.accession.release.io.VariantMongoAggregationReader.STUDY_ID_KEY;
+import static uk.ac.ebi.eva.accession.release.io.VariantMongoAggregationReader.SUBMITTED_VARIANT_VALIDATED_KEY;
+import static uk.ac.ebi.eva.accession.release.io.VariantMongoAggregationReader.SUPPORTED_BY_EVIDENCE_KEY;
+import static uk.ac.ebi.eva.accession.release.io.VariantMongoAggregationReader.VARIANT_CLASS_KEY;
+
```

Comment:
Please consider using [this](https://www.ebi.ac.uk/panda/jira/browse/EVA-1220?focusedCommentId=246718&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-246718) as test-case fodder for this test class